src/components/Builder/Newsletter/Newsletter.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+const NewsLetter = ({ parent }) => {
+  return (
+    <div className="Newsletter">
+      <a href={parent}>Optional Link</a>
+    </div>
+  );
+};
+export default NewsLetter;
```

Comment:
We have a `Button` component that should be used instead of creating this NewsLetter component. This really isnt just a `Newsletter` component either, its a more dynamic component, ie it could be a link something else not just a Newsletter